### DISCUSSION TITLE
discord server

### DISCUSSION THREAD

**Discussion by tosufever** [veryengineer/collab#11] [General]
invite: https://discord.gg/Ckx6EqGA

it's a blank slate. i'm no discord expert. if anyone wants to help set the thing up properly, lmk. 

github almost needs a livechat feature. something between twitter/reddit and discussions. idk if the answer is discord, but i've got some stupid questions that don't really belong on twitter or in discussions.

**Comment by FierceSkit** [comment]
Invite expired as of now ://

**Comment by IndigoNakamoto** [comment]
Invite still expired

**Reply by daank-c** [reply]
https://discord.gg/aPwgptWE
